ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ConsistencyOffsetVectorFunctionalTest.java

```diff
@@ -223,6 +225,7 @@ public void shouldRoundTripCVWhenExecutePullQuery() throws Exception {
     verifyConsistencyVector(serializedCV);
   }
 
+  @Ignore
```

Comment:
In `7.2.x`, `ConsistencyOffsetVectorFunctionalTest` does not exist. Please exercise caution when choosing to ignore the tests.